app/helpers/application_helper.rb

```diff
@@ -265,6 +247,20 @@ def view_to_url(view, parent = nil)
     end
   end
 
+  def view_to_association(view, parent)
+    case view.db
+    when "OrchestrationStackOutput", "OrchestrationStackParameter", "OrchestrationStackResource"
+                                       then view.db.split(/(?=[A-Z])/).last.downcase.pluralize
```

Comment:
@chessbyte I agree, I'd rather we have separate when statements for each one.

These non-standard view -> association names should really stick out so we can really understand why they're different.  In the original code, we broke the convention here and there's no explanation why... I'm sure this and other conditionals would just disappear if we stuck to conventions.